src/main/java/ch/njol/skript/expressions/ExprCommand.java

```diff
@@ -43,26 +44,28 @@
 @Name("Command")
 @Description("The command that caused an 'on command' event (excluding the leading slash and all arguments)")
 @Examples({"# prevent any commands except for the /exit command during some game",
-		"on command:",
-		"\tif {game::%player%::playing} is true:",
-		"\t\tif the command is not \"exit\":",
-		"\t\t\tmessage \"You're not allowed to use commands during the game\"",
-		"\t\t\tcancel the event"})
-@Since("2.0")
+	"on command:",
+	"\tif {game::%player%::playing} is true:",
+	"\t\tif the command is not \"exit\":",
+	"\t\t\tmessage \"You're not allowed to use commands during the game\"",
+	"\t\t\tcancel the event"})
+@Since("2.0, INSERT VERSION (Support for the unknown command event)")
 @Events("command")
 public class ExprCommand extends SimpleExpression<String> {
+	private final static int FULL = 0, LABEL = 1, ARGS = 2;
+	private final static boolean UNKNOWN_COMMAND_SUPPORTED = Skript.classExists("org.bukkit.event.command.UnknownCommandEvent");
+	
 	static {
 		Skript.registerExpression(ExprCommand.class, String.class, ExpressionType.SIMPLE,
-				"[the] (full|complete|whole) command", "[the] command [label]", "[the] arguments");
+			"[the] (full|complete|whole) command", "[the] command [label]", "[the] arguments");

```

Comment:
Each pattern should be put on it's own line IMO. Keep the two tabs too unless your IDE pushes back too hard 😝